lib/offsite_payments/integrations/world_pay.rb

```diff
@@ -34,8 +34,13 @@ class Helper < OffsitePayments::Helper
         mapping :customer, :email => 'email',
                            :phone => 'tel'
 
-        mapping :billing_address, :zip => 'postcode',
-                                  :country  => 'country'
+        mapping :billing_address,
+          :address1 => 'address1',
+          :address2 => 'address2',
+          :city => 'town',
+          :state => 'region',
+          :zip => 'postcode',
+          :country  => 'country'
```

Comment:
Before it was doing `lookup_country_code(params[:country])`. I can't remember what we set to country but we have to ensure it's the ISO country code and not the country name.

Reply:
Just checked, we should be sending the `country_code` alright!

Reply:
The `add_address` takes care of this so it's all good 👍